verify.sh

```diff
@@ -0,0 +1,12 @@
+#!/usr/bin/env bash
```

Comment:
good question. i think my assumption was that jenkins was essentially already behaving that way. is that not true for scripts like this?